update/Pos/Update/Poll/Modifier.hs

```diff
@@ -1,14 +1,55 @@
 -- | Functions working with PollModifier.
 
 module Pos.Update.Poll.Modifier
-       ( modifyPollModifier
+       (
+         -- * Poll modifier
+         PollModifier (..)
+       , pmBVsL
+       , pmAdoptedBVFullL
+       , pmConfirmedL
+       , pmConfirmedPropsL
+       , pmActivePropsL
+       , pmSlottingDataL
+       , pmEpochProposersL
+
+       , modifyPollModifier
        ) where
 
+import           Universum
+
+import           Control.Lens (makeLensesFor)
 import           Data.Default (Default (def))
 import           Data.Semigroup (Semigroup)
-import           Universum
 
-import           Pos.Update.Poll.Types (PollModifier (..))
+import           Pos.Core.Types (ApplicationName, BlockVersion, BlockVersionData,
+                                 NumSoftwareVersion, SoftwareVersion, StakeholderId)
+import           Pos.Core.Update (UpId)
+import           Pos.Slotting.Types (SlottingData)
+import           Pos.Update.Poll.Types (BlockVersionState, ConfirmedProposalState, ProposalState)
+import           Pos.Util.Modifier (MapModifier)
+
+-- | PollModifier is used in verification. It represents operation which
+-- one should apply to global state to obtain result of application of
+-- MemPool or blocks which are verified.
+data PollModifier = PollModifier
```

Comment:
Why did you move it here?

Reply:
Because when I was moving it to `core` this datatype shouldn't be moved and I decided it should be logical split with other datatypes. Also, I find it more logical to put it here.

Reply:
No, we put types into `Types` modules and other stuff into neighbour modules.